backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/llvm/IrToBitcode.kt

```diff
@@ -47,6 +46,7 @@ internal val IrField.storageClass: FieldStorage get() {
     // TODO: Is this correct?
     val annotations = correspondingPropertySymbol?.owner?.annotations ?: annotations
     return when {
+        annotations.hasAnnotation(KonanFqNames.canBePrecreated) -> FieldStorage.SHARED
```

Comment:
How does `@CanBeCreated` work on fields? What is it supposed to mean?

Reply:
Nothing, just incorrect place for the check, will fix.